parse-webidl.js

```diff
@@ -282,6 +282,7 @@ function parseType(idltype, idlNames, externalDependencies, contextName) {
                           "RegExp", "Error", "DOMException", "ArrayBuffer", "DataView", "Int8Array", "Int16Array", "Int32Array", "Uint8Array", "Uint16Array", "Uint32Array", "Uint8ClampedArray", "Float32Array", "Float64Array",
                           "ArrayBufferView", "BufferSource", "DOMTimeStamp", "Function", "VoidFunction"];
     if (wellKnownTypes.indexOf(idltype.idlType) === -1) {
+            console.error(idltype);
```

Comment:
That seems like a leftover from debug. If not, the message should be more explicit.